.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Nightly Docs
+
+on:
+    push:
+        branches:
+            - master
+            - 'dev/**'
```

Comment:
I don't believe dev/ should be an event trigger. If a dev branch gets a push it'll update to a dev build and not master.